### DISCUSSION TITLE
All Checks have Failed

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#5624] [Help]
### Summary

![failed](https://github.com/vercel/community/assets/157824398/20943564-f5a5-4f26-a92c-9ef02ed5aa3a)

I get this after pushing to repo.  There is no additional information in GitHub.  There are no logs in Vercel or information on what is causing it.  

### Steps to Reproduce

Unable to provide specific information on how to reproduce because I'm not getting any additional information from systems.  I am able to build locally with no issues.

**Comment by devashish2024** [comment]
Happens to me too, I need help. 
I guess to reproduce this: make a github account and make vercel with it, change username of github and it should happen
My all last 2 repos i tried to deploy (new projects) failed they works tho

> btw is there python 3.10 on vercel

**Reply by swarnava** [reply]
Hi, we don't support Python 3.10 yet. This is something we are working on.

**Reply by devashish2024** [reply]
Ok, but I gave how to reproduce it. Try it out. I had to delete my vercel account and it appears to be fixed, now theres a issue cuz my code is for python 3.10 only

**Comment by swarnava** [comment]
> There are no logs in Vercel or information on what is causing it.

What happens if you click on `Details`?

**Comment by devashish2024** [comment]
> ### Summary
> ![failed](https://private-user-images.githubusercontent.com/157824398/300085315-20943564-f5a5-4f26-a92c-9ef02ed5aa3a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODgwNDcsIm5iZiI6MTcwOTg4Nzc0NywicGF0aCI6Ii8xNTc4MjQzOTgvMzAwMDg1MzE1LTIwOTQzNTY0LWY1YTUtNGYyNi1hOTJjLTllZjAyZWQ1YWEzYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMwOFQwODQ5MDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mOTU1YzM3ODQ4NGUyYTZjZGMzYTU5Mzk5YzVjZDVhZjI2ZDM2ODUzNGVjMjAwYzA3YTgzYTJlODdhMGNlYWRmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.m6rv4IiufTxtyAqduRo36FPhOvEyVFsMx9LFmXTNq8I)
> 
> I get this after pushing to repo. There is no additional information in GitHub. There are no logs in Vercel or information on what is causing it.
> 
> ### Example
> _No response_
> 
> ### Steps to Reproduce
> Unable to provide specific information on how to reproduce because I'm not getting any additional information from systems. I am able to build locally with no issues.

You should go to vercel dashbaord -> project -> deployments (latest one) -> logs tab and it provides the log
